### DISCUSSION TITLE
Cannot find release binary files.

### DISCUSSION THREAD

**Discussion by aleksandr1592** [input-leap/input-leap#1628] [Q&A] [answered]
### What happened?

Is there any way to download pre-build release .exe files for windows? "Releases" link seems broken. 
![изображение](https://user-images.githubusercontent.com/8572349/228275721-79d3bf6f-c05a-4222-b083-a382469b2dd7.png)

### Version

v2.0.0-RC1

### What OSes are you seeing the problem on? (Check all that apply)

Windows

### What OS versions are you using?

Windows 11

**Comment by shymega** [comment]
This should be a discussion, not an issue. Converting. Please do not open issues for questions, especially questions that can be answered on IRC...

**Comment by shymega** [comment, marked as answer]
To follow-up, please check GH Actions for Windows builds. I do not advise installing from CI yet, because we're refactoring heavily and it's not advisable to use on a day-to-day basis.

**Comment by githamster** [comment]
Hello all,
I am real pleased that you took on his project, and I am hopeful to use it.
I only found this comment after I tried for a good two hours to build on Windows from Cl, trying to debug workting with QT6 etc. :0)
Never used GitHub Actions, and I can not for the life of me figure out how to use them to create a binary.
Could someone be so kind and write a lil guide here so that we can make use of the actions, and try the builds and feed back?
If it works, I'd rather donate some $ to you guys than to ShareMouse ;-)

Many thanks!

**Reply by shymega** [reply]
You don't need to code for GitHub Actions, we're refactoring CI, so this may change, but for example, the latest build from the main branch, is [here](https://github.com/input-leap/input-leap/suites/11995461709/artifacts/629775024). This is built on Windows Server 2019, and is an EXE installer, I *think*, I don't use Windows full-time.
